src/jobs/deploy-lambda.yml

```diff
@@ -1,9 +1,59 @@
 description: >
   This job deploys an application to aws lambda.
 
+parameters:
+  aws-region:
```

Comment:
Put required parameters at the top. (those that dont have a default value) and add a comment separating required and non required